src/node.cc

```diff
@@ -543,7 +543,7 @@ void RegisterSignalHandler(int signal,
   struct sigaction sa;
   memset(&sa, 0, sizeof(sa));
   sa.sa_sigaction = handler;
-  sa.sa_flags = reset_handler ? SA_RESETHAND : 0;
+  sa.sa_flags = reset_handler ? SA_RESETHAND | SA_SIGINFO : SA_SIGINFO;
```

Comment:
```suggestion
  sa.sa_flags = SA_SIGINFO | (reset_handler ? SA_RESETHAND | 0);
```

Would be a bit more readable and compact I think.

Reply:
done!